libs/java/dal/db/dal-db/src/main/java/ca/bc/gov/tno/dal/db/AuditColumns.java

```diff
@@ -58,10 +61,11 @@
    * When the record was last updated. Automatically set by the DB. Provides
    * concurrency control to enforce optimistic concurrency.
    */
-  @Version
+  // @Version
```

Comment:
Had to disable because Java can't handle serialization/deserialization of Date...